python/build_definitions/protobuf.py

```diff
@@ -21,8 +21,9 @@
 class ProtobufDependency(Dependency):
     def __init__(self) -> None:
         super(ProtobufDependency, self).__init__(
-            'protobuf', '3.5.1',
-            'https://github.com/google/protobuf/releases/download/v{0}/protobuf-cpp-{0}.tar.gz',
+            'protobuf',
+            '3.5.1-yb-1',
+            'https://github.com/yugabyte/protobuf/archive/refs/tags/v{0}.tar.gz',
```

Comment:
It would be nice to see a bit more rationale for YB-specific version os protobuf, either here or in the yugabyte-fork of that repo.

Reply:
Explained in the description.